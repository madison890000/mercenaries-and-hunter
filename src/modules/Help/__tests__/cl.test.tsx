import React, {PropsWithChildren} from 'react';
import renderer from 'react-test-renderer';
import CL from "../cl";
import {BrowserRouter as Router} from "react-router-dom";

const RouterWrapper: React.FC<PropsWithChildren> = ({children}) => {
    return <Router>{children}</Router>;
};
test('renders CL', () => {
    const WithPaddingLayoutDom = renderer.create(
        <RouterWrapper>
            <CL/>
        </RouterWrapper>
    ).toJSON();
    expect(WithPaddingLayoutDom).toMatchSnapshot();
});
