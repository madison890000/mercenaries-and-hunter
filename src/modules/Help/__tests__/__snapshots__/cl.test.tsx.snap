// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CL 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-bhp9pd-MuiPaper-root-MuiCard-root"
  style={
    Object {
      "background": "var(--color-blue-9)",
      "margin": 30,
      "padding": 30,
      "textAlign": "left",
    }
  }
>
  <div
    className="ant-row ant-row-space-between css-dev-only-do-not-override-byeoj0"
    style={Object {}}
  >
    <div
      className="ant-col css-dev-only-do-not-override-byeoj0"
      style={Object {}}
    >
      <h2
        style={
          Object {
            "marginLeft": "20px",
          }
        }
      >
        AI Cover Letter
      </h2>
      <div
        style={
          Object {
            "fontStyle": "italic",
            "margin": 20,
            "textAlign": "left",
          }
        }
      >
        <div>
          投递没回应，没有写Cover Letter？
        </div>
        <div>
          只是换名称，重复Cover Letter？
        </div>
        <div>
          抓耳挠骚，不知如何写Cover Letter？
        </div>
      </div>
      <div
        style={
          Object {
            "fontWeight": "bolder",
            "margin": 20,
            "textAlign": "left",
          }
        }
      >
        <div>
          一键生成，告别蹩脚的求职信！
        </div>
        <div
          style={
            Object {
              "marginTop": 20,
            }
          }
        >
          <button
            className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            快去体验
          </button>
        </div>
      </div>
    </div>
    <div
      className="ant-col css-dev-only-do-not-override-byeoj0"
      style={Object {}}
    >
      <img
        height="320"
        src="/gif/cl.gif"
      />
    </div>
  </div>
</div>
`;
